lib/active_support/cache/database_store.rb

```diff
@@ -26,16 +26,33 @@ def initialize(options = nil)
       end
 
       # Preemptively iterates through all stored keys and removes the ones which have expired.
+
+      # params [Hash] options
+      # option options [String] :namespace
+      # option options [ActiveSupport::Duration] :created_before - provide a time duration after record without expire_at date will get removed.
       def cleanup(options = nil)
         options = merged_options(options)
         scope = @model.expired
+
+        scope = if (created_before = options[:created_before])
```

Comment:
sorry, I think this should be:

```rb
if (created_before = options[:created_before])
  scope = scope.or(@model.created_before(created_before))
end
```

Reply:
we should not remove have this scope applied by default

Reply:
@dim I have adjusted code to not use `.created_before` scope by default. But I, personally, feel that there should be a sane default here.